0001-kaa-protocol/README.md

```diff
@@ -99,6 +99,9 @@ Thus, for them the resource paths start with:
 
 The rest of the resource path is extension-specific and is described in separate documents that define KP protocol extensions.
 
+### MQTT QoS
+KP uses MQTT QoS level 0.
```

Comment:
KP does not use QoS level 0 only. The client is allowed to use any QoS level (if the server supports that).

If you still want to define the minimum supported QoS level for KP-compliant implementation, that should be QoS2 (because it won't be MQTT-compliant without that).

So what I propose is just ignore the question. It is implicit that the server _should_ support all QoS levels.